docker-compose/startup.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash       
+
+docker info
```

Comment:
Out of interest, why is this file a specific script but for Helm it's inline in the travis file?

Reply:
I've been trying to get it in also but build is failing. I'll try again tomorrow morning.

Reply:
It's absolutely fine it being a separate file, but we should be consistent.

Either everything is inline or everything is a script across helm and compose. Also, if it's a script we should probably put it in the ./travis folder to show it's specific to the build and not something a customer should try and run.

Reply:
fixed all scripts are in .travis.yml now